ptx/sec_deriv_intro.ptx

```diff
@@ -1230,7 +1317,14 @@
           </caption>
           <!-- START figures/fig_piecewisecosx1.tex -->
           <image xml:id="img_piecewisecosx1" width="47%">
-            <description></description>
+            <shortdescription>
+              Graph of derivative of function that is <m>\cos(x)</m> for values of <m>x \leq \pi/2</m> and <m>x \gt \pi/2</m>.
+            </shortdescription>
+            <description>
+              <p>
+                Graph of derivative of the function that exists at <m>\fp(\pi/2) </m> and follows a curve of <m>\cos(x)</m> for <m>x\leq\pi/2</m> and <m>0</m>.
```

Comment:
The long description needs more detail here. There are some important features: one is that cos(pi/2)=0, so the derivative is continuous at pi/2.  But there is a sharp bend in the graph at pi/2, suggesting that the function is not differentiable there.